proto/generate.proto

```diff
@@ -249,7 +249,7 @@ message Generation {
     /// Request ID
     uint64 request_id = 1;
     /// Prefill tokens (optional)
-    PrefillTokens prefill_tokens = 2;
+    NextTokens prefill_tokens = 2;
```

Comment:
Should we just remove the `PrefillTokens` proto from here if it's not used anywhere?